scripts/docker/pom-modify/PomFile.java

```diff
@@ -447,6 +484,9 @@ public static void main(String[] args) {
             // First create objects out of all the pom.xml files passed in
             while ((input = bufReader.readLine()) != null) {
                 PomFile p = new PomFile(input);
+		if (testrunnerPluginExists){
```

Comment:
As mentioned in the comment above, maybe the warning can be here to just print a message that no changes are done here due to the plugin already existing.

Reply:
Ok, done. The extraction of the idflakies version number in the pom file was a bit painful and I am sure there must be better ways to analyze XML documents... Modularizing this large code block would certainly benefit the code structure and understandability. I'll leave this for future work and more talented/experienced Java coders.